CONTRIBUTING.md

```diff
@@ -110,7 +110,7 @@ For feature work, take a look at the current [Enhancements](https://github.com/a
 
 If it has someone assigned to it then that person is the person responsible for working the enhancement. If you feel like you could contribute then reach out to that person.
 
-Fixing bugs, adding translations, and updating the documentation are always appreciated, so reviewing the backlog of issues is always a good place to start. For extra information on debugging tools, see [Debugging](https://github.com/ansible/awx/blob/devel/docs/debugging.md).
+Fixing bugs, adding translations, and updating the documentation are always appreciated, so reviewing the backlog of issues is always a good place to start. For extra information on debugging tools, see [Debugging](./docs/debugging/README.md).
```

Comment:
Would this link work if it were to point to `./docs/debugging/` (the directory) or does it have to be a file?

Reply:
Yes, this also would work when linking to a directory. I understood your question as a wish to change it, and so I did. I amended my commit, and the link is now pointing to `./docs/debugging/`. And because there's a `README.md` present in that directory, it automatically gets printed below the directory listing.